lib/errors.ts

```diff
@@ -149,6 +149,8 @@ const EXPECTED_ERROR_REGEXES = [
 	/^BalenaDeviceNotFound/, // balena-sdk
 	/^BalenaExpiredToken/, // balena-sdk
 	/^BalenaInvalidDeviceType/, // balena-sdk
+	/Cannot deactivate devices/i, // balena-sdk
+	/Devices must be offline in order to be deactivated.$/, // balena-sdk
```

Comment:
Case insensitive matching for this one as well?

Reply:
yeah yeah let me check that one, manually testing it now

Reply:
ok works fine